packages/react/src/components/Chat/ChatMessage.tsx

```diff
@@ -142,12 +143,24 @@ class ChatMessage extends UIComponent<WithAsProp<ChatMessageProps>, ChatMessageS
     isFromKeyboard: false,
   }
 
+  messageRef = React.createRef<HTMLElement>()
+
   actionHandlers = {
     // prevents default FocusZone behavior, e.g., in ChatMessageBehavior, it prevents FocusZone from using arrow keys
     // as navigation (only Tab key should work)
     preventDefault: event => {
       event.preventDefault()
     },
+
+    focus: event => {
+      if (this.messageRef) {
+        this.messageRef.current.focus()
+
+        if (document.activeElement === this.messageRef.current) {
+          event.stopPropagation()
```

Comment:
why is this needed? can you maybe add a comment in the code as well as it is not so obvious.

Reply:
Yep, this combination looks a bit strange... Is `.focus()` async?

Reply:
it's more about stop propagation only if focus was set to chat message successfully, for e.g. if chat message won't have tabindex and it is a `div`, focus will go to body. If we allow propagation, most probably it can be handled by parent components.

Ok, discussed with Shift, this is very rare case. I'll just remove it to avoid concerns